templates/details/libraries/docstring.html

```diff
@@ -4,18 +4,46 @@
 
 {% block libraries_content %}
 
-<div id="docstrings" data-js="tab-content" class="p-docstring">
+<div class="p-docstring">
   <div class="p-docstring__intro">
-    {{ docstrings['html'] | safe }}
+    <div class="p-strip is-shallow u-no-padding--top">
+      {{ docstrings['html'] | safe }}
+    </div>
+    <hr class="p-separator--dense">
+  </div>
+  <div class="row p-strip is-shallow u-no-padding--top">
+    <div class="col-1">
+      <p class="p-text--x-small-capitalized">Index</p>
+    </div>
+    <div class="col-7">
+      <ul class="p-list">
+        {% for group in docstrings['content'] %}
+        {% if not group["name"].startswith("__") %}
+        <li class="p-list__item">
+          <a href="#{{ group['name'].lower() }}">{% if group["type"].lower() == "function" %}def {{ group["name"] }}(){% else %}{{ group["type"].lower() }} {{ group["name"] }}{% endif %}</a>
+        </li>
+        {% endif %}
+        {% for item in group["content"] %}
+        <li class="p-list__item u-no-padding">
+          <ul class="p-list u-no-margin--bottom" style="margin-inline-start: 1.5rem;">
+            {% if not item["name"].startswith("__") %}
+            <li class="p-list__item">
+              <a href="#{{ group['name'].lower() }}-{{ item['name'].lower() }}">{% if item["type"].lower() == "function" %}def {{ item["name"] }}(){% else %}{{ item["type"].lower() }} {{ item["name"] }}{% endif %}</a>
+            </li>
+            {% endif %}
+          </ul>
+        </li>
+        {% endfor %}
```

Comment:
It would be more readable with more indentation here

Reply:
agree, will fix it